src/dbnode/client/session.go

```diff
@@ -129,7 +129,7 @@ var (
 	errUnableToEncodeTags = errors.New("unable to include tags")
 	// errNoTopology is returned when the session does not have a topology. Should never happen
```

Comment:
nit: update comment to `errNoTopologyMap`

Reply:
done